src/libcommon/comm.h

```diff
@@ -119,6 +119,7 @@ enum class RequestNum {
     UPDATER_SKIP_VERSION,
     UTILITY_CRASH,
     UTILITY_QUIT,
+    CLIENT_STARTED
```

Comment:
Rename to UTILITY_ REPORT_CLIENT_IS_VISIBLE ?

Reply:
Done here : [64cf668](https://github.com/Infomaniak/desktop-kDrive/pull/403/commits/64cf6687950e2de1f4467ed10393ee66ec39d45c) & [4e1c6f8](https://github.com/Infomaniak/desktop-kDrive/pull/403/commits/4e1c6f8d23c98c19ad4fdd4a96ac07a2e0933eef)

Reply:
OK